src/main/java/commands/Command.java

```diff
@@ -0,0 +1,21 @@
+package commands;
+
+import data.TaskList;
+import data.exception.DukeException;
+import storage.Storage;
+import ui.Ui;
+
+/**
+ * The Command abstract class.
+ * 
+ * Represents the logic that is to be executed
+ * for the respective user command.
+ */
+public abstract class Command {
+    public abstract void execute(
+            TaskList tasks, Storage storage, Ui ui) throws DukeException;
+
+    public boolean isExit() {
+        return false;
+    }
```

Comment:
The isExit() method can be further documented to explain its use case. Is it to determine if a command should lead to an exit?